### PR TITLE
Amendment's made to previous PR 

### PR DESCRIPTION
**Pull request by aFuzzyBear**
Here as the amended changes to the original PR #76 , 

Please advise further if there is anything else that might need fixing,

Thanks =)

### CODE REVIEW

File: public/index.css [hunk 1]
```diff
@@ -4,27 +4,18 @@
 }
 
 :root {
-  --user-font-scale: 1rem - 16px;
-  --max-width: calc(100% - 2rem);
+  --user-font-scale: calc(1em+1vw);
```
Comment (FredKSchott) [inline public/index.css:7]: @natemoo-re can you review these styling changes? Since you originally wrote them I'm not sure what this changes. @aFuzzyBear it would be great if you could add a comment (in this GH thread is fine) mentioning why these changes were needed

**Comment by FredKSchott** [comment]
I think your branch is behind a bit, and some things have changed! Can you rebase off of main (or merge with main) to resolve conflicts, and then resubmit?

Alternatively, it could be good to break apart the "content" markdown changes from the CSS/code changes, into two separate PRs. It's easier to review and merge a markdown PR if there are no code changes.

File: public/index.css (hunk 1 above)
Comment (aFuzzyBear) [reply public/index.css:7]: This wee liner is a bit of typography css trickary i had picked up to make the font-size responsive to the viewport, the em bit is the size of the letters as it bottoms out and the vw is the size of the letters respective to the viewport, It makes the typography alot more fluid and respective to the different viewports alot easier,

https://bits.theorem.co/css-pro-tips-responsive-font-sizes-and-when-to-use-which-units/
https://allthingssmitty.com/2016/12/05/flexible-type-using-root/

**Comment by aFuzzyBear** [comment]
Sorry @FredKSchott , Ill do that from now on
